financial-connections/src/main/java/com/stripe/android/financialconnections/presentation/FinancialConnectionsSheetNativeViewModel.kt

```diff
@@ -493,6 +494,7 @@ internal class FinancialConnectionsSheetNativeViewModel @Inject constructor(
                     .initialSyncResponse(args.initialSyncResponse.takeIf { state.firstInit })
                     .application(app)
                     .configuration(state.configuration)
+                    .sharedComponent(FinancialConnectionsSingletonSharedComponentHolder.getComponent(app))
```

Comment:
Not used yet, but later in the flow this will allow us to generate tokens using the same reference instantiated on the previous activity.